### PR TITLE
FNX-14546 ⁃ For #13096: Add notifications pref in top level settings

### PR DESCRIPTION
**Pull request by eliserichards**
For #13096 

image

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by eliserichards** [comment]
@brampitoyo, it looks like app notification settings were only added in Oreo (api 26). What should we do for the lower api levels? Route to SUMO?

**Comment by Mugurell** [comment]
On Fennec, for Android versions lower than Oreo we just didn't show this setting.

**Comment by eliserichards** [comment]
Confirmed with @brampitoyo that we'll follow Fennec's impl :)

File: app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -269,6 +269,22 @@ class SettingsFragment : PreferenceFragmentCompat() {
             resources.getString(R.string.pref_key_delete_browsing_data_on_quit_preference) -> {
                 SettingsFragmentDirections.actionSettingsFragmentToDeleteBrowsingDataOnQuitFragment()
             }
+            resources.getString(R.string.pref_key_notifications) -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                    val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS)
+                    startActivity(intent)
+                } else {
+                    (activity as HomeActivity).openToBrowserAndLoad(
```
Comment (Mugurell) [inline app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt:277]: Do we still need the if/else if the notification settings are only shown for >=Oreo devices?

**Pull request merged**